src/main/useCases/ComputePasswordStrength.java

```diff
@@ -0,0 +1,20 @@
+package useCases;
```

Comment:
Java naming conventions should be package use_cases

Reply:
Agreed. Done!